src/util/Factory.sol

```diff
@@ -124,16 +124,17 @@ contract RestrictionManagerFactory is Auth {
         emit Rely(msg.sender);
     }
 
-    function newRestrictionManager(
-        uint8, /* restrictionSet */
-        address token,
-        address[] calldata restrictionManagerWards
-    ) public auth returns (address) {
+    function newRestrictionManager(uint8 restrictionSet, address token, address[] calldata restrictionManagerWards)
```

Comment:
`restrictionSet` is an unused param for now right? That's why I commented the name